lms/templates/student_account/account_settings_section.underscore

```diff
@@ -0,0 +1,18 @@
+<% _.each(sections, function(section) { %>
+    <div class="section">
+        <% if (section.subtitle) { %>
+            <h4 class="account-settings-header-subtitle"><%- gettext(section.subtitle) %></h4>
+        <% } %>
+        <h3 class="section-header"><%- gettext(section.title) %></h3>
+        <div class="account-settings-section-body">
+            <div class="ui-loading-indicator">
+                <span class="spin"><i class="icon fa fa-refresh" aria-hidden=true></i></span>
```

Comment:
We are using span elements here instead of i's now: http://ux.edx.org/design_elements/icons/